pelicanconf.py

```diff
@@ -216,6 +216,12 @@ def __init__(self, url, title, context, css="", children=()):
         "discourse": "crisclacerda",
         "tagline": "GSoC 2020 Student",
     },
+    "Aanyu Deborah Oduman": {
+        "github": "deborahtrez",
```

Comment:
The author pic is not working. Did you run `tools/update_avatars.py` and add/commit the image to the repo?

Reply:
Done it now.